DataConnectors/CiscoFirepower/CiscoFirepowerEStreamerCollector.json

```diff
@@ -39,8 +39,7 @@
         }
     ],
     "availability": {
-        "status": 1,
-        "isPreview": true
+        "status": 1
```

Comment:
isPreview tag is missing. Please add the isPreview tag.

Thanks,
Aditi Maheshwari